tests/test_configuration.py

```diff
@@ -86,7 +86,12 @@ def test_dict(self):
             "sym": False,
             "spectral_indexing": "ansi",
             "surface": np.array(
-                [[0, 0, 0, 10, 0], [0, 1, 0, 1, 1], [0, -1, 1, 0.1, 0.1]]
+                [
+                    [0, 0, 0, 10, 0],
+                    [0, 1, 0, 1, 0],
+                    [0, -1, 0, 0, -1],
+                    [0, -1, 1, 0.1, 0.1],
+                ]
```

Comment:
these were changed because the previous values gave a degenerate solution with zero volume